dozer/cogs/teams.py

```diff
@@ -0,0 +1,85 @@
+
+from .. import db
+from ._utils import *
+import discord
+from discord.ext.commands import BadArgument
+
+
+class Teams(Cog):
+	@command()
+	async def setteam(self, ctx, team_type, team_number):
+		"""Sets an association with your team in the database."""
+		team_type = team_type.casefold()
+		with db.Session() as session:
+			dbtransaction = TeamNumbers(user_id=ctx.author.id, team_number=int(team_number), team_type=team_type)
+			session.add(dbtransaction)
+		await ctx.send("Team number set!")
+	setteam.example_usage = """
+	`{prefix}setteam type team_number` - Creates an association in the database with a specified team
```

Comment:
For the team type, do we have a standard system for this? I understand most people say "FRC" and "FTC" but what about VRC? Won't some people say vex or something else? We should have the accepted types in the example usage and check if it's in there.